docs/release-notes/v1.50-v1.59.md

```diff
@@ -2,13 +2,310 @@
 
 !!!info "Target users"
 
-    **Curators** - users who add new data into ODM and are responsible for the data harmonization and curation. That includes creating and defining metadata templates, mapping metadata and temples, and data updates.
+    **Curators** - users responsible for adding new data to ODM and ensuring data harmonization and curation. While curators typically do not have permission to define, create, or edit metadata templates, they can select existing templates to use for their data. Their responsibilities include mapping metadata, associating data with templates, and performing data updates.
 
-    **Researchers** - users who access ODM to identify a batch of data suitable for further research and analysis. That includes search, data browsing and export.
+    **Researchers** - users who access ODM to identify and retrieve data suitable for further research and analysis. Their activities include searching, browsing, and exporting data.
 
-    **Advanced** - users who can utilize advanced API functionalities for user management and data management.
+    **Advanced** - uusers with access to advanced API functionalities, such as user management and data management operations.
 
-    **Admins** - users who manage organization in ODM, its user and groups.
+    **Admins** - users who manage the organization in ODM, including overseeing users, groups, and permissions. They are also responsible for creating, defining, and editing metadata templates as needed.
+
+## 1.59 Open Data Manager 2024 Winter Release
+
+This release focuses on optimizing workflows with expanded filtering options for VCF files, streamlined reference genome management, and improved API-based file management, all of which increase flexibility and precision in data handling.
+
+### New Features
+
+#### [Researchers]
+
+* **Enhancements for VCF functionality: Filtering variant data based on genes: variantFeature**
+
+Starting with release 1.59, ODM streamlines the process of retrieving gene variants within a specified gene region by introducing the variantFeature parameter. Instead of manually specifying chromosome and interval information from the original VCF file, users can now simply provide a gene name. The system automatically references the associated genome to determine the intervals, reads the file, and returns the relevant data, making the process faster and more user-friendly.
+
+* ??? abstract "variantFeature considerations"
+    * Multiple values are provided as a string array.
+    * Gene(s) should be supplied (e.g. feature=TP53 )
+    * The system searches for **exact matches** for the supplied gene(s) in the reference genomes.
+    * Each gene is resolved to a region for each Reference genome in the system.
+
+* ??? danger "The **variantFeature** parameter has been added to the following endpoints"
+    * GET /api/v1/as-user/variants
+    * GET /api/v1/as-curator/variants
```

Comment:
In this section and the one below (regarding filtering by the INFO field), we should also mention that this functionality is supported in vxQuery and affects all omics endpoints with this parameter. I’m unsure of the best way to organize this; perhaps it would be better to create a dedicated section summarizing all the changes in vxQuery, as its syntax is slightly different. I’ll leave the decision up to you.

Reply:
Done